### ISSUE TITLE
JSON Results Collapse ALL

### ISSUE THREAD

**Issue by christatedavies** [a85/POSTMan-Chrome-Extension#1257]
I'd love it if there was a shortcut to collapse all the nodes in the body response (for JSON results)

Cheers!

**Comment by johnnavarra** [comment]
you should be able to click these: 
![image](https://cloud.githubusercontent.com/assets/11891698/8701124/f90490be-2add-11e5-9303-0da3d4f5878e.png)

**Comment by christatedavies** [comment]
Thanks, but I am looking for a shortcut to collapse ALL in one click.

**Comment by andrewduffy77** [comment]
+1 it's killing me!

**Comment by codedmart** [comment]
I would love to see this feature as well.

**Comment by golubitsky** [comment]
+1!

**Comment by Nexen23** [comment]
I'm really dissapointed. Without this feature it is almost useless for me. I will use something else than :(

**Comment by nategranatir** [comment]
+1 this makes a big difference when you get a huge response and want to traverse by node. without collapse all, for larger responses I have to copy/paste the response into another viewer.

**Comment by ms88privat** [comment]
:+1:

**Comment by ankitj** [comment]
+1

**Comment by aguetat** [comment]
+1

**Comment by dtw45** [comment]
+1

**Comment by bilalshahid10** [comment]
+1

**Comment by sebrandon1** [comment]
+1

**Comment by gotwilk890** [comment]
+1

**Comment by voxon2** [comment]
+1

**Comment by andreykats** [comment]
+1

**Comment by abhijitkane** [comment]
@andreykats @voxon2 @gotwilk890 @sebrandon1 @bilalvirgo10 @dtw45 @aguetat @ankitj @ms88privat 
You can use `Alt-0 (Win)    / Command-Option-0 (Mac)` for 'fold all', and `Alt-Shift-0 (Win) /  Command-Option-Shift-0 (Mac)` for 'Unfold all'.
The focus will have to be on the response JSON/XML, though.

**Comment by andreykats** [comment]
@abhijitkane Amazing! Let's put this in the documentation.

**Comment by abhijitkane** [comment]
Yep. All AceEditor shortcuts (https://github.com/ajaxorg/ace/wiki/Default-Keyboard-Shortcuts) should work.

**Comment by nategranatir** [comment]
@abhijitkane you're a hero

**Comment by wzup** [comment]
@abhijitkane 

(https://github.com/postmanlabs/postman-app-support/issues/1257#issuecomment-227872031):
> You can use Alt-0 (Win).....

Is there an option to have a JSON response first level fields collapsed by default? In order to make less keyboard clicks to collapse them

**Comment by ernestojpg** [comment]
That's cool, but I never remember the shortcut. Why isn't there an easy to see button, or an option in a contextual menu for doing it? :)

**Comment by dotancohen** [comment]
Note that to unfold a portion of JSON with `Alt-Shift-L` (See abhijitkane's comment for link), the cursor must be on the *folded marker*, not simply on the line.

**Comment by sargin48** [comment]
+1 Collapse all / expand all xml json

**Comment by ActionScripted** [comment]
+1 Collapse/expand all toggles.

Would be great to have a larger inline expand/collapse toggle, too, similar to how Firefox handles JSON.

**Comment by asbachb** [comment]
I guess an option to collapse a json response in general would be quite useful for some people.

**Comment by aaroncrawford** [comment]
And if we want to get real crazy, an option to fold per level would be fantastic.  Something like CMD+OPT 1 folds to level 1, CMD+OPT+2 folds level 2 etc.

**Comment by dotancohen** [comment]
@aaroncrawford : You mean Ctrl-Alt-1, Ctrl-Alt-2 :)
Yes, I agree that would be absolutely terrific. Most of the time we are interested in the actual _response_, and a good way to navigate the response would be very helpful.

**Comment by shamasis** [comment]
This has already been added. We need to have this documented and perhaps even highlighted as contextual in-product documentation. 🤦‍♂ Will close the ticket when the doc goes live.

**Comment by clemkake** [comment]
Ah finally a good answer @shamasis 
I was hoping to see that!

**Comment by MrSankofa** [comment]
If you looking at this in 2024, the shortcuts have changed and command + option + 0 no longer works for mac users

it's now

command + k + 0 to collapse all
command + K + J to open


https://community.postman.com/t/command-option-0-wont-collapse-all/11861
